src/DynamoCore/Nodes/NodeCategories.cs

```diff
@@ -224,12 +224,12 @@ internal static string ShortenCategoryName(string fullCategoryName)
             if (string.IsNullOrEmpty(fullCategoryName))
                 return string.Empty;
 
-            var catName = fullCategoryName.Replace(Configurations.CategoryDelimiter.ToString(), " " + Configurations.ShortenedCategoryDelimiter + " ");
+            var catName = fullCategoryName.Replace(Configurations.CategoryDelimiterString.ToString(), " " + Configurations.ShortenedCategoryDelimiterString + " ");
```

Comment:
It seems like `CategoryDelimiterString` is already a `string`, do we need `ToString` here?